pkg/pb/piecestore.proto

```diff
@@ -49,6 +49,7 @@ message PieceStore {
 
   RenterBandwidthAllocation bandwidthallocation = 1;
   PieceData piecedata = 2;
+  SignedMessage signedMessage = 3;
```

Comment:
According to the [protobuf style guide](https://developers.google.com/protocol-buffers/docs/style) this should be named like `signed_message` instead of `signedMessage`.

I know this protobuf file does not follow the style guide closely, but let's add the new things with the proper style. Perhaps, we can fix the rest in another PR.

Reply:
done, changed to `authorization`